server/www/templates/posts/detail.html

```diff
@@ -1,125 +1,147 @@
-                        <p sd-model-to-html="post.content"></p>
-                    </div>
-                    <div
-                        ng-if="!!post.tags.length"
-                        ng-show="visibleStage == stages[0].id || visibleStage == 'post'"
-                        class="form-field"
-                    >
-                        <legend class="input-label" for="tags" ng-class="{ 'error': form.tags.$invalid && form.tags.$dirty }" translate>post.modify.form.categories</legend>
-                        <div class="form-field checkbox" ng-repeat="category in post.tags">
-                            <label>
-                                <i class="fa fa-{{category.icon}}" style="color:{{category.color}}"></i> {{category.tag}}
-                            </label>
-                        </div>
-                    </div>
-
-                    <div
-                        ng-repeat="(key,value) in post.values"
-                        ng-if="form_attributes[key] && showType(form_attributes[key].type)"
-                        ng-show="form_attributes[key].form_stage_id == visibleStage"
-                        class="post-field"
-                      >
-
-                       <!-- Show images -->
-                      <post-media-value
-                        ng-if="form_attributes[key].type === 'media'"
-                        label="{{form_attributes[key].label}}"
-                        media-id="value"></post-media-value>
-
-                      <post-value
-                        key="key"
-                        value="value"
-                        attribute="form_attributes[key]"
-                        ng-if="form_attributes[key].type !== 'media'">
-                        ></post-value>
-                    </div>
-                    <div
-                        ng-if="mapDataLoaded"
-                        ng-show="visibleStage == stages[0].id || visibleStage == 'post'"
-                        class="post-field"
-                        >
-                        <h4 ng-if="mapDataLoaded" translate>post.location</h4>
-                        <div ng-if="mapDataLoaded">
-                            <leaflet id="post-map" class="map" defaults="defaults" geojson="geojson" center="center" layers="layers"></leaflet>
-                          </div>
-                    </div>
-                </div>
+<div class="mode-context" ng-controller="navigation as nav">
+
+  <header class="mode-context-header has-logo">
+    <h1 class="mode-context-title"><a href="/" ng-bind="nav.site.name"></a></h1>
+    <img ng-if="site.image_header" ng-src="nav.site.image_header" class="deployment-logo" />
+  </header>
+
+  <span class="mode-context-trigger">
+    <svg class="iconic">
+      <use xlink:href="../img/iconic-sprite.svg#chevron-bottom"></use>
+    </svg>
+    <span class="label hidden">Show more/less</span>
+  </span>
+
+  <div class="mode-context-body">
+    <p ng-bind="nav.site.description"></p>
```

Comment:
I don't think site description is what we're supposed to be showing here.. check with @brandonrosage but I'd guess it should be the form description..